sdk/confidentialledger/azure-confidentialledgertools-receiptverification/tests/_shared/constants.py

```diff
@@ -0,0 +1,186 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from azure.confidentialledgertools.receiptverification.models import (
+    LeafComponents,
+    ProofElement,
+    Receipt,
+)
+
+
+def get_test_valid_service_certificate_1():
+    return """
+-----BEGIN CERTIFICATE-----
```

Comment:
These look like public keys; I don't think you need to add the file to exclude from secrets scanning.